packages/web-components/src/components/ic-select/ic-select.e2e.ts

```diff
@@ -1053,6 +1133,33 @@ describe("ic-select", () => {
       expect(menuOptions[3]).toEqualText("Macchiato");
     });
 
+    //TODO: Uncomment when bug is fixed
```

Comment:
has this bug been fixed, or is it a separate ticket?

Reply:
This is a seperate ticket that has been pulled into sprint (https://github.com/mi6/ic-ui-kit/issues/704). I left a note on that issue telling whoever picks this up to uncomment this when that ticket is done